### ISSUE TITLE
AutoGraph warning when using auto-keras

### ISSUE THREAD

**Issue by D-Katt** [tensorflow/tensorflow#51431]
I was running auto-keras StructuredDataRegressor in Kaggle notebook, when this warning appeared:

WARNING: AutoGraph could not transform function <function Model.make_train_function.<locals>.train_function at ... and will run it as-is.
Please report this to the TensorFlow team.
Cause: closure mismatch, requested ('self', 'step_function'), but source function had ()

The search for regression model seams to be working, but the warning appears at every trial.

This is the dataset I was using: https://www.kaggle.com/c/tabular-playground-series-aug-2021

And this is the code:

!pip install autokeras
import pandas as pd
import numpy as np
from sklearn.preprocessing import QuantileTransformer
from sklearn.model_selection import train_test_split
import tensorflow as tf
import autokeras as ak

TRAIN_PATH = '../input/tabular-playground-series-aug-2021/train.csv'
TARGET_NAME = 'loss'
VAL_SIZE = 0.2

data_train = pd.read_csv(TRAIN_PATH)

X = data_train.drop('id', axis='columns')
y = X.pop(TARGET_NAME)

scaler = QuantileTransformer(output_distribution='normal')
X = scaler.fit_transform(X)

groups = pd.cut(
    y, bins=10, labels=[1, 2, 3, 4, 5, 6, 7, 8, 9, 10])

x_train, x_valid, y_train, y_valid = train_test_split(
    X, y, stratify=groups, test_size=VAL_SIZE,
    shuffle=True, random_state=0)

model = ak.StructuredDataRegressor(
    overwrite=False, max_trials=100)

model.fit(x_train, y_train, epochs=1,
          validation_data=(x_valid, y_valid))

**Comment by sushreebarsa** [comment]
@D-Katt Please post this issue on [keras-team/keras repo.](https://github.com/keras-team/keras/issues)
To know more see;
[https://discuss.tensorflow.org/t/keras-project-moved-to-new-repository-in-https-github-com-keras-team-keras/1999](https://discuss.tensorflow.org/t/keras-project-moved-to-new-repository-in-https-github-com-keras-team-keras/1999)
Thank you!

**Comment by D-Katt** [comment]
Posted there.

**Comment by sushreebarsa** [comment]
@D-Katt Thank you for the update! Could you please move this issue to closed status as you have posted it in Keras repo , you will get the right help there.Thanks!
